jib-core/src/main/java/com/google/cloud/tools/jib/api/Containerizer.java

```diff
@@ -239,7 +239,8 @@ public Containerizer setAllowInsecureRegistries(boolean allowInsecureRegistries)
 
   /**
    * Sets whether or not to run the build in offline mode. In offline mode, the base image is
-   * retrieved from the cache instead of pulled from a registry.
+   * retrieved from the cache instead of pulled from a registry, and the build will fail if the base
+   * image is not in the cache.
```

Comment:
Javadoc nit: I believe it will always fail when pushing to a registry (when using `RegistryImage`) too, right?